src/index.test.js

```diff
@@ -128,5 +107,59 @@ describe('ActiveStorageProvider', () => {
         )
       })
     })
+
+    describe('if no X-CSRF-Token is provided', () => {
```

Comment:
In this unit test for the `ActiveStorageProvider` component, I'd prefer not to test the implementation of `csrfHeader`. The reason for mocking it out with an arbitrary value (line 16) is to test that it is correctly delegated to. You know? For all inputs, test (only) your return values and your side effects. We don't need to go through the options for what it returns, since that's the concern of its own unit tests. But of course, the fact that it can be overridden by a custom headers prop is definitely something that `ActiveStorageProvider` is doing, so we need to test that.

Reply:
I hear what you're saying. I left this test to make sure props were correctly being sent to csrfHeader